app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -206,21 +206,22 @@ def load_errata_applications(filter_errata_type: nil, include_last_reboot: 'yes'
         search_up_to = up_to.present? ? "ended_at < \"#{up_to}\"" : nil
         search_since = since.present? ? "ended_at > \"#{since}\"" : nil
         search_result = status.present? && status != 'all' ? "result = #{status}" : nil
-        search = [search_up_to, search_since, search_result].compact.join(' and ')
+        labels = 'label ^ (Actions::Katello::Host::Erratum::Install, Actions::Katello::Host::Erratum::ApplicableErrataInstall)'
+        select = 'foreman_tasks_tasks.*'
 
         if Katello.with_remote_execution?
-          condition = ["state = 'stopped' AND ((label = 'Actions::RemoteExecution::RunHostJob' AND templates.id = ?) " \
-                       "OR label = 'Actions::Katello::Host::Erratum::Install' OR label = 'Actions::Katello::Host::Erratum::ApplicableErrataInstall')",
-                       RemoteExecutionFeature.feature('katello_errata_install').job_template_id]
+          new_labels = 'label = Actions::RemoteExecution::RunHostJob AND remote_execution_feature.label = katello_errata_install'
```

Comment:
The remote execution feature can be either `katello_errata_install` or `katello_errata_install_by_search`. The former is used on the old pages, and `katello_errata_install_by_search` is used on the new host detail page.

I'm surprised no one caught this sooner..

Reply:
Good catch, added